extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -74,4 +75,14 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 
 		return vscode.Uri.file(projectFile);
 	}
+
+	/**
+	* Adds the list of files and directories to the project, and saves the project file
+	* @param projectFile The Uri of the project file
+	* @param list list of file and folder paths to add
+	*/
+	async addToProject(projectFile: vscode.Uri, list: string[]): Promise<void> {
```

Comment:
shouldn't the items to be added also of type Uri?

Reply:
I can change it to Uri if that makes more sense. Right now, only createProjectFromDatabase is also using this to add files and folders.

Reply:
for consistency, I'd say let's make it Uri too